modules/nw-networkpolicy-optimize-ovn.adoc

```diff
@@ -0,0 +1,98 @@
+// Module included in the following assemblies:
+//
+// * networking/network_policy/about-network-policy.adoc
+
+[id="nw-networkpolicy-optimize-ovn_{context}"]
+= Optimizations for network policy with OpenShift OVN
+
+When designing your network policy, refer to the following guidelines:
+
+* For network policies with the same `spec.podSelector` spec,  it is more efficient to use one network policy with multiple `ingress` or `egress` rules, than multiple network policies with subsets of `ingress` or `egress` rules.
+
+* Every `ingress` or `egress` rule based on the `podSelector` or `namespaceSelector` spec will generate the number of OVS flows proportional to `number of pods selected by network policy + number of pods selected by ingress or egress rule`. Therefore, it is preferable to use the `podSelector` or `namespaceSelector` spec that can select as many pods as you need in one rule, instead of creating individual rules for every pod.
++
+For example, the following two rules:
++
+[source,yaml]
+----
+apiVersion: networking.k8s.io/v1
+kind: NetworkPolicy
+metadata:
+ name: test-network-policy
+spec:
+  podSelector: {}
+  ingress:
+    - from:
+       - podSelector:
+           matchLabels:
+             role: frontend
+    - from:
+       - podSelector:
+           matchLabels:
+             role: backend
+----
++
+Can be expressed by only one rule:
```

Comment:
I'm not thrilled with the _For example, the following two rule can be expressed with one network policy._ sentence being split like this. It works, but it's weird. 

Maybe:

> The following policy contains two rules:
> The following policy expresses those same two rules as one: